_components/search-bar.md

```diff
@@ -8,55 +8,40 @@ title: Search Bar
 
   <div class="usa-grid">
     <div class="usa-width-one-half">
-      <form class="usa-search usa-search-big">           
-        <fieldset>
```

Comment:
I thought we didn't need fieldsets for a single field? I thought that was even what you told me. Anyway, here is what @nickbristow said: 

"its basically when you need to group fields with a title. or like, question answer, so question is a legend for a fieldset of radio buttons"

Here's what Sasha said:
"I am pretty sure as long as the label is associated with the input then no"